# Generated for NaoQi method ALBehaviorManager.stopAllBehaviors
# Stop all behaviors

---


